deepfence_server/pkg/df-error/registry.go

```diff
@@ -0,0 +1,7 @@
+package df_error
```

Comment:
Instead of creating new package  can this be moved to deepfence_utils/errors so we can define all common errors in one single place in future

Reply:
sounds good!

Reply:
pushed!